packages/adblocker/src/filters/network.ts

```diff
@@ -536,24 +678,26 @@ export default class NetworkFilter implements IFilter {
             }
 
             // Ignore this filter if no redirection resource is specified
-            if (optionValue.length === 0) {
+            if (value.length === 0) {
               return null;
             }
 
+            mask = setBit(mask, NETWORK_FILTER_MASK.isRedirect);
+
             if (option === 'redirect-rule') {
               mask = setBit(mask, NETWORK_FILTER_MASK.isRedirectRule);
             }
 
-            redirect = optionValue;
+            optionValue = value;
```

Comment:
Why is this `optionValue` in some cases and `value` in others? Is this `optionValue` the new global optional string for redirect, etc.?

Reply:
We decided to use `optionValue` as a shared property in `NetworkFilter` to address all modifiers with option of long string. This decision helps us to reduce the overall size of `NetworkFilter` when serialized and keep the structure simpler with less properties in total.